package.json

```diff
@@ -8,7 +8,7 @@
     "postinstall": "node installer/setup.js"
   },
   "engines": {
-    "node": ">=8"
+    "node": ">=14"
```

Comment:
In the `index.js` file you told it to throw an error if the version wasn't v12, yet here you have told it to require v14 or greater, please fix for consistency.

Reply:
ok